src/Entity/Contact.php

```diff
@@ -54,29 +55,29 @@
 class Contact extends DonneePersonnelle
 {
     #[ORM\Column(name: 'prenom', type: 'string', length: 255, nullable: false)]
-    #[Groups(['read-personal-data', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
+    #[Groups(['read-personal-data', 'read-personal-data-v2', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
     #[Anonymize('fr-fr.firstname')]
     #[Assert\NotBlank]
     private ?string $prenom = null;
 
     #[ORM\Column(name: 'telephone', type: 'string', length: 255, nullable: true)]
-    #[Groups(['read-personal-data', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
+    #[Groups(['read-personal-data', 'read-personal-data-v2', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
     #[Anonymize('fr-fr.phone')]
     private ?string $telephone = null;
 
     #[ORM\Column(name: 'email', type: 'string', length: 255, nullable: true)]
-    #[Groups(['read-personal-data', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
+    #[Groups(['read-personal-data', 'read-personal-data-v2', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
     #[Anonymize('string', options: ['sample' => [AnonymizationHelper::ANONYMIZED_EMAIL]])]
     #[Assert\Email]
     private ?string $email = null;
 
     #[ORM\Column(name: 'commentaire', type: 'text', length: 0, nullable: true)]
-    #[Groups(['read-personal-data', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
+    #[Groups(['read-personal-data', 'read-personal-data-v2', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
     #[Anonymize('string', options: ['sample' => [AnonymizationHelper::ANONYMIZED_CONTENT]])]
     private ?string $commentaire = null;
 
     #[ORM\Column(name: 'association', type: 'string', length: 255, nullable: true)]
-    #[Groups(['read-personal-data', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
+    #[Groups(['read-personal-data', 'read-personal-data-v2', 'write-personal-data', 'v3:contact:write', 'v3:contact:read'])]
```

Comment:
Why did you have to create a new serialization group and not use the existing one (read-personal-data) ?